### ISSUE TITLE
Option to customize date format

### ISSUE THREAD

**Issue by iznaut** [mProjectsCode/obsidian-media-db-plugin#78]
**Is your feature request related to a problem? Please describe.**
I'm trying to do some querying using the Dataview plugin but it seems to have trouble parsing the date values from this plugin.

Here's an example:

```dataview
table releaseDate
sort releaseDate asc
where releaseDate > date(2018-01-01)
```

<img width="738" alt="Screenshot 2022-12-19 at 6 43 35 PM" src="https://user-images.githubusercontent.com/36773942/208555344-5b49a5bf-f31b-41c3-ba1d-b8eea76d2e1c.png">

- The first entry has date format `YYYY-MM-DD` (updated manually). It appears as expected.
- The second entry has date format `MM/DD/YYYY` (automatically set via Media DB plugin). It should be filtered out.
- The third entry is expected to be there, but it's also worth noting that the date doesn't get nicely formatted like it does in the first entry.

**Describe the solution you'd like**
A config option for preferred date format to be used when creating new Media DB entries.

**Additional context**
I acknowledge that this may also be something that could be fixed on the Dataview side of things (or more likely - if I made an advanced JS query it would be possible already to handle this format), but since Obsidian's Daily Note feature already uses `YYYY-MM-DD` by default, I don't think it's unreasonable to think this plugin should support that format as well 🙂 

<img width="784" alt="Screenshot 2022-12-19 at 6 51 05 PM" src="https://user-images.githubusercontent.com/36773942/208556102-41e77295-8b0e-44d2-b16f-91dc385dbf5e.png">

**Comment by mProjectsCode** [comment]
Good idea, I will implement this if I can find some free time during the Christmas and new year vacation

**Comment by mProjectsCode** [comment]
This has long been implemented.
